charts/ccd-user-profile-api/values.yaml

```diff
@@ -2,7 +2,7 @@ java:
   applicationPort: 4453
   aadIdentityName: ccd
   environment:
-    USER_PROFILE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_definition,ccd_admin
+    USER_PROFILE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_definition,ccd_admin,rd_professional_api
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
@hemantt   / @MSancaktutar  bump version in Chart.yaml and use same in cnp-config-flux. 

Alternatively you can override param `USER_PROFILE_S2S_AUTHORISED_SERVICES`  in the flux config directly. 
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common/ccd/user-profile-api.yaml